rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -91,11 +94,14 @@
     private Coin feePerKb;
     private ABICallElection feePerKbElection;
 
+    private HashMap<DataWord, Optional<Integer>> storageVersion;
```

Comment:
For me it looks a little weird to have a map of optional values. Maybe the use of an initial version for the cases when the `Optional.empty()` is used would prevent readability things like `storageVersion.get(key).isPresent()` vs `storageVersion.containsKey(key)`

Reply:
I think that that is not a minor change and the functionallity is the same, i would merge this as is and then look for a cleaner approach later on.